Author: jasaw


diff --git a/src/ffmpeg.c b/src/ffmpeg.c
index 3dfd966..1ed4c01 100644
--- a/src/ffmpeg.c
+++ b/src/ffmpeg.c
@@ -721,7 +721,7 @@ static const char *ffmpeg_codec_is_blacklisted(const char *codec_name)
             * - remove the "h264_omx" from this blacklist.
             * More information: https://github.com/Motion-Project/motion/issues/433
             */
-            {"h264_omx", "Codec causes lock up on your FFMpeg version"},
+            //{"h264_omx", "Codec causes lock up on your FFMpeg version"},
     #endif
     #if ( MYFFVER < 57041)
             {"h264_v4l2m2m", "FFMpeg version is too old"},
